src/darwin/Framework/CHIP/MTRDeviceController.h

```diff
@@ -173,7 +173,7 @@ typedef void (^MTRDeviceConnectionCallback)(MTRBaseDevice * _Nullable device, NS
  * Attempts to retrieve the attestation challenge for a commissionee with the given Device ID.
  * Returns nil if given Device ID does not match an active commissionee, or if a Secure Session is not availale.
  */
-- (nullable NSData *)fetchAttestationChallengeForDeviceID:(uint64_t)deviceID MTR_NEWLY_AVAILABLE;
+- (NSData * _Nullable)fetchAttestationChallengeForDeviceID:(NSNumber *)deviceID MTR_NEWLY_AVAILABLE;
```

Comment:
If we're changing the selector anyway we could consider removing the `fetch...`, it doesn't really seem to add anything to the meaning here (and we're only returning some in-memory state synchronously rather than really "fetching" anything from the device it sounds like)

Reply:
Happy to do that.  So just `attestationChallengeForDeviceID`?

Reply:
Done.